### DISCUSSION TITLE
Can anyone share the grafana template shown in this Readme.md? Also, is there a discussion group like wechat/QQ/telegram/discord?

### DISCUSSION THREAD

**Discussion by kz2318nkd** [chaos-mesh/chaos-mesh#2515] [General]
![image](https://user-images.githubusercontent.com/56473340/141259009-5cc3102a-8902-4736-9ffd-8611c67d399b.png)

**Comment by kz2318nkd** [comment]
In fact, I just want a monitoring tool to view the impact of the highly available chaos experiment on the pod container and the experimental results. The effect in the figure above is good. Who can help provide it? Thank you!

**Reply by STRRL** [reply]
Hi @kz2318nkd, I am afraid that we could not share the original dashboard with you. Actually that dashboard was created with the grafana datasource plugin, https://github.com/chaos-mesh/datasource. You need to install it (with the certain version with Chaos Mesh)  manually, we still waiting on the review from grafana official plugin marketplace. 


We have a slack channel: https://cloud-native.slack.com/archives/C0193VAV272;

And we have a wechat group. You could add me as a contact and I would invite you to join.  My wechat id is `strrlthedev`

**Reply by kz2318nkd** [reply]
Thanks for your answer, I will study this grafana plugin.
My slack email is _kz2318nkd@gmail.com_,I hope I can join.In addition,I have added your wechat and noted my github ID, I hope it can be approved!
